tests/locale/keymap_or_locale.pm

```diff
@@ -10,99 +10,26 @@
 # Summary: Keyboard layout test in console and display manager after boot
 # Maintainer: Oliver Kurz <okurz@suse.de>
 
-use base "opensusebasetest";
+use base "locale";
 use strict;
-use warnings;
-use testapi;
-use utils;
 use Utils::Backends 'has_ttys';
+use testapi 'assert_screen';
+use utils 'ensure_serialdev_permissions';
 
-sub verify_default_keymap_textmode_non_us {
-    my ($test_string, $tag) = @_;
-    # Installation with different keyboard layout is not feature ready but
-    # in case of autoyast scenarios we can simply test on login prompt without changing tty
-    type_string $test_string;
-    assert_screen ["${tag}", "${tag}_not_ready"];
```

Comment:
@Soulofdestiny seems like you miss https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/eb4ddf125a152c0899a95034641da8c6feafde6d#diff-824c96311858d5ae7f03f7bb6a5c9e5c now